synfig-studio/src/synfigapp/actions/layerlowermax.cpp

```diff
@@ -0,0 +1,157 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file layerlowermax.cpp
+**	\brief Template File
+**
+**	\legal
+**	Copyright (c) 2002-2005 Robert B. Quattlebaum Jr., Adrian Bentley
+**	Copyright (c) 2008 Chris Moore
+**
+**	This file is part of Synfig.
+**
+**	Synfig is free software: you can redistribute it and/or modify
+**	it under the terms of the GNU General Public License as published by
+**	the Free Software Foundation, either version 2 of the License, or
+**	(at your option) any later version.
+**
+**	Synfig is distributed in the hope that it will be useful,
+**	but WITHOUT ANY WARRANTY; without even the implied warranty of
+**	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+**	GNU General Public License for more details.
+**
+**	You should have received a copy of the GNU General Public License
+**	along with Synfig.  If not, see <https://www.gnu.org/licenses/>.
+**	\endlegal
+*/
+/* ========================================================================= */
+
+/* === H E A D E R S ======================================================= */
+#ifdef USING_PCH
+#	include "pch.h"
+#else
+#ifdef HAVE_CONFIG_H
+#	include <config.h>
+#endif
+
+#include <synfig/general.h>
+
+#include "layerlowermax.h"
+#include "layermove.h"
+#include <synfigapp/canvasinterface.h>
+
+#include <synfigapp/localization.h>
```

Comment:
Let's sort these `#include`:

First the direct header: "layerlowermax.h"

Then we group from base libraries to the directly related:

glib
gdk
gtk
etl (bye)
synfig
synfigapp
synfigapp/actions

and finally, on each group, alphabetically.

Reply:
Done